index.html

```diff
@@ -48,6 +48,8 @@ <h1>Join us at PyCon</h1>
           <p>Whether you are an experenced programmer, a hobby hacker or an absolute beginner, we'd love to welcome you to the Python community.</p>
           <p>PyCons are hosted all around the world, mostly organized by volunteers from local Python communities.</p>
           <p>This means that each PyCon is unique, although all PyCons share the same spirit &mdash; to spread knowledge, learn from each other and make lasting connections.</p>
+          <p>If you are looking to start a new PyCon, you'll need to contact the <a href="https://www.python.org/psf/committees/#trademarks-work-group" style="color: white">Python Software Foundation trademark workgroup</a> for permission. Please confirm that: 1.) there is no other regional PyCon in your area (e.g. country) 2.) the PyCon has a Code of Conduct and 3.) the PyCon is a community conference and not for commercial purposes.
```

Comment:
I'd be curious how this would render with an actual `<ol>` instead of an inline, and difficult to parse, list.

Reply:
Ah yes, let me do that! 🤦